setup.py

```diff
@@ -16,7 +16,7 @@
     'entry_points': {
         'console_scripts': ['rosversion=rospkg.rosversion:main'],
     },
-    'install_requires': ['catkin_pkg', 'PyYAML'],
+    'install_requires': ['catkin_pkg', 'PyYAML', 'distro'],
```

Comment:
Nitpick: please insert in alphabetical order.

Please also add the dependencies for the Debian package in the `stdeb.cfg` file.